### PR TITLE
[NUI] Implement certificate & authentication APIs of WebView.

### PR DESCRIPTION
**Pull request by huayongxu**
This patch is to implement ssl certificate & http authentication
of WebView.

### API Changes ###

 - ACR:

### CODE REVIEW

**Comment by huayongxu** [comment]
This patch is related to the patch at csharp-binder side:
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/254283/

**Comment by dongsug-song** [comment]
looks good to me, but other member should review also.

@huayongxu 
could you resolve the error below?
**"This branch has conflicts that must be resolved"**

File: src/Tizen.NUI/src/internal/WebView/WebView.cs [hunk 1]
```diff
@@ -1180,5 +1282,26 @@ private void OnFrameRendered(IntPtr data)
         {
             frameRenderedEventHandler?.Invoke(this, new EventArgs());
         }
+
+        private void OnCertificateConfirmed(IntPtr data, IntPtr certificate)
+        {
+            WebCertificate webCertificate = new WebCertificate(certificate, false);
```
Comment (huiyueun) [inline src/Tizen.NUI/src/internal/WebView/WebView.cs:1288]: How about changing this?
It does not create unnecessary objects.
```csharp
certificateConfirmedEventHandler?.Invoke(this, new WebViewCertificateReceivedEventArgs(new WebCertificate(certificate, false)));
```

Comment (huayongxu) [reply src/Tizen.NUI/src/internal/WebView/WebView.cs]: Done

**Comment by dongsug-song** [comment]
@huayongxu 
could you resolve the conflict?

**Comment by huayongxu** [comment]
> @huayongxu
> could you resolve the conflict?

Done.

**Review by AdunFang — APPROVED** [review]
It is OK.

**Review by zhouleonlei — APPROVED** [review]
LGTM

**Review by AchoWang — APPROVED** [review]
Looks good to me.

**Review by elishateng — APPROVED** [review]
LGTM

**Review by zg2nets — APPROVED** [review]
lgtm

**Review by rabbitfor — APPROVED** [review]
LGTM

**Pull request merged**